frameworks/JavaScript/0http/app.js

```diff
@@ -1,25 +1,26 @@
-const cluster = require('cluster')
-const cpus = require('os').cpus()
-
 const cero = require('0http')
-const { router, server } = cero()
+const low = require('0http/lib/server/low')
+
+const { router, server } = cero({
+  server: low()
+})
+
+const date = new Date().toUTCString()
```

Comment:
You either have to have some mechanism that updates this on the server every second or so or generates this date in the handlers for each route.